src/app/static/src/app/store/baseline/actions.ts

```diff
@@ -27,7 +27,9 @@ export const actions: ActionTree<BaselineState, RootState> & BaselineActions = {
             .freezeResponse()
             .postAndReturn<PjnzResponse>("/baseline/pjnz/", formData)
             .then(() => {
-                dispatch('metadata/getPlottingMetadata', state.iso3, {root: true});
+                if (!state.baselineError && state.iso3) {
+                    dispatch('metadata/getPlottingMetadata', state.iso3, {root: true});
+                }
                 dispatch('validate');
```

Comment:
I guess we don't really need to validate either since we know there's nothing to validate against - have left as is but could change this.